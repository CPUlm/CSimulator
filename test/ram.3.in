7
01
1
10
1010
10
1
10
0110
10
1
00
0100
01
1
01
1100
11
1
11
0001
01
1
11
0000
11
1
11
0010
