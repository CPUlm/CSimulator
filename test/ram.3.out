Step 1:
=> o = 0000
Step 2:
=> o = 1010
Step 3:
=> o = 1110
Step 4:
=> o = 0000
Step 5:
=> o = 0000
Step 6:
=> o = 1100
Step 7:
=> o = 1100
